frontend/packages/ui/src/Modal.tsx

```diff
@@ -19,25 +21,39 @@ export interface ModalProps {
    */
   onClose?: () => void
   /**
-   * Boolean value that defines if close button should be displayed
+   * Title of the modal. Default is 'Details'
    */
-  showCloseButton?: boolean
+  title?: string
 }
 
 export const Modal: FunctionComponent<ModalProps> = ({
   isOpen = false,
   onClose,
-  showCloseButton = true,
+  title = 'Details',
   ...props
 }) => {
   return isOpen ? (
     <Container>
       <Backdrop />
       <Content gap={2}>
-        {showCloseButton && (
-          <CloseButton variant="outline" size="medium" onClick={onClose} />
-        )}
+        <Header>
+          <Text type="text xl" bold={true}>
+            {title}
+          </Text>
+          <Icon
+            name="close"
+            size="large"
+            color="black"
+            className="c-icon"
+            onClick={onClose}
+          />
+        </Header>
         {props.children}
+        <PageLayout.Footer height={4.5}>
```

Comment:
We can put 
props.children in ModalBody that has a padding, something like this:
 <ModalBody>{props.children}</ModalBody>
 
 Also probably my mistake I said it wrong. We should have something similar to PageLayout.Footer but not to use it. Just have the same thing here, so that it's optional for developer to put it outside of this component like this: 
```
<Modal title="Some header">
Some content
<Modal.Footer>
<Button>Cancel</Button>
</Modal.Footer>
</Modal>
```

Reply:
Modal body with padding added.

I added optional footer, but could not replicate how pageLayout works.